packages/tools/pie-wrapper-react/scripts/add-react-wrapper.js

```diff
@@ -0,0 +1,113 @@
+import { createWriteStream } from 'fs';
+import path from 'path';
+import fs from 'fs-extra';
+
+let componentSrc;
+
+const componentPath = path.resolve(process.cwd(), './custom-elements.json');
+
+// fetches custom-elements.json file
+const loadJSON = (file) => JSON.parse(fs.readFileSync(file));
+
+/**
+ * This function generates a react wrapper to enable custom lit components to be used in react apps.
+ *
+ * @param {JSON} - A JSON file of custom components and their attributes, generated from the @custom-elements-manifest/analyzer package
+ * @param {string} - component folder name fetched from package.json script
+ * @return {string} - The source code of the react wrapper
+ *
+ */
+export function addReactWrapper (customElementsObject, folderName = process.argv[2]) {
+    const components = [];
+    const customElements = Object.entries(customElementsObject);
+
+    // sort through customElements array and put all components into a separate array
+    customElements.forEach(([key, value]) => {
+        let componentObject;
+        if (key === 'modules') {
+            value.forEach((k) => {
+                if (k.path.includes(folderName)) {
+                    k.declarations.forEach((decl) => {
+                        componentObject = decl.customElement === true ? components.push({ class: decl, path: k.path.replace('index.js', 'react.ts') }) : '';
```

Comment:
Looks like `componentObject` is not needed and this could become a simple if statement.

`if(decl.customElement === true) components.push({ class: decl, path: k.path.replace('index.js', 'react.ts') });`